### ISSUE TITLE
postponeEnterTransition() does not work in "Picasso Fragment to Fragment" sample.

### ISSUE THREAD

**Issue by aartikov** [mikescamell/shared-element-transitions#8]
Calling of a method postponeEnterTransition() has no effect. To check it remove startPostponedEnterTransition() calls in a Picasso callback. A transition will be still plays, but it should not.  Documentation says: "Likewise, postponement only works if FragmentTransaction optimizations are enabled." So, to fix the sample call setAllowOptimization(true) of a FragmentTransaction.

**Comment by Rainer-Lang** [comment]
@aartikov Do you want to make a PR?

**Comment by aartikov** [comment]
@Rainer-Lang No, i don't. It is looks like that this repository is abandoned.

**Comment by mikescamell** [comment]
@aartikov It's not abandoned, I'm just pretty busy at the moment. Feel free to make a PR.

**Comment by aartikov** [comment]
@mikescamell done #11

**Comment by mikescamell** [comment]
@aartikov I'm going to need to look into this more as turning the optimisation on has an effect on the Shared Element Transitions.
